bp_common/syn/Makefile.vcs

```diff
@@ -40,8 +40,20 @@ VCS_BUILD_OPTS += +plusarg_save
 VCS_BUILD_OPTS += $(BP_TOOLS_DIR)/install/lib/libdromajo_cosim.a
 VCS_BUILD_OPTS += -top testbench
 VCS_BUILD_OPTS += -f flist.vcs
-
-VCS_PLUSARGS  =
+ifdef UVM
+VCS_BUILD_OPTS += -ntb_opts uvm-1.2 +incdir+$(BP_UVM_DIR)
```

Comment:
incdir can be done in uvm/flist.vcs

Reply:
This is now done in the most recent commit.